docs/modules/pbmarkdup.md

```diff
@@ -0,0 +1,15 @@
+---
+Name: pbmarkdup
+URL: https://github.com/PacificBiosciences/pbmarkdup
+Description: >
+  pbmarkdup takes one or multiple sequencing chips of an amplified libray as
+  HiFi reads and marks or removes duplicates.
+---
+
+The pbmarkdup module parses results generated by
+[pbmarkdup](https://github.com/PacificBiosciences/pbmarkdup), and adds the
+**% Unique Molecules** and **%Duplicate Reads** to the General Statistics
+table.
+
+This module assumes that pbmarkdup has been run for every sample, and that
+the pbmarkup log file is named `sample.pbmarkdup.log`.
```

Comment:
Removed now that you updated the search criteria.

```suggestion
This module assumes that pbmarkdup has been run for every sample.
```

I don't totally understand what you mean by being run for every sample. It should handle summaries that contain more than one sample in them, right?

Reply:
I've removed this line, since I'm not sure why I added it. pbmarkdup runs on single samples.